website/docs/r/cis_mtls.html.markdown

```diff
@@ -0,0 +1,65 @@
+---
+subcategory: "Internet services"
+layout: "ibm"
+page_title: "IBM: ibm_cis_cache_settings"
+description: |-
+  Provides an IBM Mutual TLS resource.
+---
+
+# ibm_cis_mtls
+ Provides mutaul TLS(MTLS) settings resource. The resource allows to create, update, or delete cache settings of a domain of an IBM Cloud Internet Services CIS instance. For more information about mtls, see [CIS MTLS](https://cloud.ibm.com/docs/cis?topic=cis-mtls-features).
+
+## Example usage
+
+```terraform
+# Change MTLS setting of the domain
+
+resource "ibm_cis_mtls" "mtls_settings" {
+  cis_id             = data.ibm_cis.cis.id
+  domain_id          = data.ibm_cis_domain.cis_domain.domain_id
+  cert_mtls       = "-----BEGIN CERTIFICATE----- -----END CERTIFICATE-----"
+  cert_name       = "MTLS_Cert"
+  host_name       = "abc.abc.abc.com"
+}
+```
+
+## Argument reference
+
+Review the argument references that you can specify for your resource. 
+
+- `cis_id` - (Required, String) The ID of the IBM Cloud Internet Services instance.
+- `domain_id` - (Required, String) The ID of the domain to change cache settings.
+- `cert_mtls` - (Required, String) Content of valid MTLS certificate.
+- `cert_name` - (Required, String) Name of certificate. 
+- `host_name` - (Required, String) Valid host name for which we want to add the certificate.
+
+**Note**
+
+Among all the purge actions `purge_all`, `purge_by-urls`, `purge_by_hosts`, and `purge_by_tags`, only one is allowed to give inside a resource.
+
+## Attribute reference
+In addition to all argument reference list, you can access the following attribute reference after your resource is created.
+
+- `id` - (String) The record ID. It is a combination of `<domain_id>,<cis_id>` attributes concatenated with `:`.
+
+## Import
+The `ibm_cis_mtls` resource can be imported using the ID. The ID is formed from the domain ID of the domain and the CRN concatenated  using a `:` character.
```

Comment:
I think ths ID of combination of three (mtlsid, domainid and crn)...make changes accordingly

Reply:
OK.